.test-infra/kubernetes/hbase/SmallITCluster/setup-all.sh

```diff
@@ -0,0 +1,43 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# This script starts HBase cluster and hbase-external service that allows to reach cluster
+# from developer's machine. Once the cluster is working, scripts waits till
+# external cluster endpoint will be available. It prints out configuration line that
+# should be added to /etc/hosts file in order to work with HBase cluster.
+#
+
+#!/bin/sh
```

Comment:
shebangs must be the first line in the file so probably just move it to the top in this in all the shell files.

Reply:
Extra points if you open a subsequent PR to fix this in all the other shell scripts (+ eventual fixes for shellcheck issues).

Reply:
Ok this makes sense. In this PR I just wanted to be consistent with current script pattern, but this could be easily fixed.